atomic_defi_design/Dex/Addressbook/AddAddressForm.qml

```diff
@@ -113,8 +113,8 @@ Dex.Rectangle
             AddressTypeSelector
             {
                 id: addressTypeComboBox
-                Layout.fillWidth: true
-                Layout.fillHeight: true
+                width: 300
+                height: 38
```

Comment:
Layout.preferredHeight

Reply:
using layout here results in `[19:03:30] [warning] [main.prerequisites.hpp:92] [4149538]: qrc:/Dex/Addressbook/AddAddressForm.qml:113:13: QML AddressTypeSelector: Binding loop detected for property "implicitHeight"`